gdal/ogr/ogrsf_frmts/cad/libopencad/dwg/r2000.cpp

```diff
@@ -1700,14 +1750,7 @@ CADSolidObject * DWGFileR2000::getSolid( long dObjectSize, CADCommonED stCommonE
 
     fillCommonEntityHandleData( solid, pabyInput, nBitOffsetFromStart );
 
-    nBitOffsetFromStart += 8 - ( nBitOffsetFromStart % 8 );
-    solid->setCRC( ReadRAWSHORT( pabyInput, nBitOffsetFromStart ) );
-
-#ifdef _DEBUG
```

Comment:
Not sure we have to remove debug msgs since I sometimes met very strange behaviour on different files (size = real_size +-1, but works just fine, but sometimes +-2 and even more), so it probably will be helpful for future improvements.